core/data_migration_scripts/EE_DMS_Core_4_1_0.dms.php

```diff
@@ -1053,7 +1053,7 @@ public function get_or_create_state($state_name, $country_name = '')
     {
         if (! $state_name) {
             throw new EE_Error(esc_html__(
-                "Could not get-or-create state because no state name was provided",
+                "Could NOT get-or-create state because no state name was provided",
```

Comment:
Leave lowercase so it doesn't break string translations?

Reply:
also